karma.conf.js

```diff
@@ -1,22 +1,72 @@
+// Karma configuration
+// Generated on Wed Dec 06 2017 14:07:02 GMT-0600 (CST)
+
 module.exports = function(config) {
   config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
     frameworks: ['mocha'],
 
+
+    // list of files / patterns to load in the browser
     files: [
-      '*.js'
+      './client/**/**/*.spec.js'
+    ],
+
+
+    // list of files to exclude
+    exclude: [
     ],
 
-    client: {
-      mocha: {
-        // change Karma's debug.html to the mocha web reporter
-        reporter: 'html',
 
-        // require specific files after Mocha is initialized
-        require: [require.resolve('bdd-lazy-var/bdd_lazy_var_global')],
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+      './client/**/*.js': ['webpack'],
+    },
+
+    webpack: require('./webpack.config.js'),
```

Comment:
`karma-webpack` requires a webpack configuration. We can inline the webpack configuration for this project here.